src/business-logic/services/database-change-ingest-service.ts

```diff
@@ -108,6 +118,12 @@ export class DatabaseChangeIngestService implements IngestChangeService {
     partChangedListener.onDeleted(partId => this.enqueueEvent(7, () => this.deletePart(partId)))
   }
 
+  private listenForMediaChanges(mediaChangedListener: DataChangedListener<Media>): void {
+    mediaChangedListener.onCreated(media => this.enqueueEvent(10, () => this.createMedia(media)))
+    mediaChangedListener.onUpdated( media => this.enqueueEvent(16, () => this.updateMedia(media)))
+    mediaChangedListener.onDeleted( mediaId => this.enqueueEvent(19, () => this.deleteMedia(mediaId)))
+  }
```

Comment:
Didn't we talk about creating a separate file for handling of the Media changes?

Reply:
Yes, i think i got a bit confused by it being an "ingest"-service, and thought of it as "things that come through ingest operations" rather than the ingested entity concept and then forgot that this doesn't belong. I'll look into it!

Reply:
It has been moved into a separate file now! The class is called `MediaDataChangeService`.

Reply:
Maybe that's a good reason to change the name of `database-change-ingest-service` to make it more descriptive and (hopefully) less confusing?